.github/workflows/create-backport-trackers.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: Create backport trackers for trackers in "Pending Backport" state
+on:
+  # To manually trigger this: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+  workflow_dispatch:
+  schedule:
+    # Every 5 minutes: https://crontab.guru/every-5-minutes
+    - cron: '*/5 * * * *'
+jobs:
+  create-backports:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Is it possible to just download the script directly? i.e.

https://raw.githubusercontent.com/ceph/ceph/main/src/script/backport-create-issue

Reply:
Yup, that's probably the best approach. I'll do. I also have to add the `requirements.txt` file so that Github caches the pip deps instead of downloading it

Reply:
Done.